src/provider/controller/ResolverHandler.ts

```diff
@@ -11,63 +11,36 @@ import {SERVICE_IDENTITY} from '../../client/internal';
 
 import {AsyncInit} from './AsyncInit';
 
-const SELECTOR_URL = (() => {
+const RESOLVER_URL = (() => {
     let providerLocation = window.location.href;
 
     if (providerLocation.indexOf('http://localhost') === 0) {
         // Work-around for fake provider used within test runner
         providerLocation = providerLocation.replace('/test', '/provider');
     }
 
-    // Locate the default selector HTML page, relative to the location of the provider
-    return providerLocation.replace('provider.html', 'ui/selector.html');
+    // Locate the default resolver HTML page, relative to the location of the provider
+    return providerLocation.replace('provider.html', 'ui/resolver');
 })();
 
 /**
- * When the user is shown an app selector popup, they have the option of telling
- * the service how to handle similar intents in the future.
- *
- * This enum defines the options available to users.
+ * Data passed to app resolver when it is invoked by the provider
  */
-export const enum DefaultAction {
```

Comment:
If we're removing the defaults stuff from the UI does it make sense to also cleanup all of the dafaults parts in the provider? (more for @pjbroadbent )

Reply:
Can you point to the parts you are referring to?

Reply:
Oh, guess that all got removed during the refactor without me noticing. Never mind then.